### DISCUSSION TITLE
Welcome to freerouting Discussions!

### DISCUSSION THREAD

**Discussion by abelom** [freerouting/freerouting#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andrasfuchs** [comment]
I'm Andras, one of the former developers of freerouting. I worked on the project only for a few months and my modifications were mostly related to command line arguments, logging, business logic / GUI separation and code quality improvements.

I know that many of our users still use freerouting so although I don't have enough free time to continue working on the code full time, I would be more than happy to share my limited knowledge of the source code and help others to create pull requests to improve freerouting further.

**Comment by f1gwr** [comment]
Hi,
Just downloaded freerouting.dmg 1.5.0, opening it under OSX Mojave 10.14.6, application says "freerouting is corrupted, should be sent to the trash". The Java counterpart .jar doesn't open either.
